client/src/Overview/StyleSelector.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
 import * as Styles from './styledComponents.js';
+import checkmark from './images/checkmark.png';
 
 const StyleSelector = ({ styles, setCurrentStyle, currentStyle }) => {
 
   return (
     <Styles.StyleSelector>
+
+      <Styles.Check className="fas fa-check checkmark"></Styles.Check>
+
       <Styles.StyleName className="styleTitle">{'Style Name > '}{currentStyle.name}</Styles.StyleName>
+
       {styles.map((style, index) => {
-        let classNames = 'style';
-        return <div id={style.name}><Styles.Style className={classNames} id={style.name} src={style.photos[0].thumbnail_url} alt={style.name + 'Style'} key={index} onClick={() => {
-          document.getElementById(style.name).style.backgroundImage = 'url(./images/checkmark.png)';
-          document.getElementById(style.name).style.zIndex = 1;
-          document.getElementById(currentStyle.name).style.backgroundImage = null;
+        return <Styles.Style className='style' id={style.name} src={style.photos[0].thumbnail_url} alt={style.name + 'Style'} key={style.name} onClick={(event) => {
```

Comment:
This is a pretty cool way to get the checkmark on the pic. I think it's a little easier to read if you define the handler function then call it onClick instead of writing it inside the brackets.